specs/agents/log-onboarding.md

```diff
@@ -0,0 +1,85 @@
+# Log Onboarding
+
+The Agents will be a critical part of log collection onboarding for their
+application logs. This is primarily accomplished via the `log_ecs_formatting`
+configuration option, described below.
+
+In future iterations, the shipping of ECS logs will become more automated by auto-parsing ECS-JSON logs in Filebeat and [automatically shipping log files](https://github.com/elastic/apm/issues/374) that got reformatted via `log_ecs_formatting`.
+
+## `log_ecs_formatting` configuration
+
+Configures the agent to automatically format application logs as ECS-compatible JSON
+(if possible).
+
+The configuration option must be marked experimental for now to allow for breaking changes we may need to introduce.
+Once the end-to-end process for seamless log onboarding with Elastic Agent works, we'll remove the experimental flag.
+
+|                |   |
+|----------------|---|
+| Valid options  | `override`, `off` (case insensitive) |
+| Default        | `off`   |
+| Dynamic        | `false` |
```

Comment:
Is the meaning of `false` the same as `off`?

Reply:
Not sure what you mean - `off` is the default value for this config. Regardless, this config option is not dynamic, hence `false` for Dynamic.

Reply:
I see - I misunderstood what
> Dynamic: `false`

means. Now it's clear. Thank you.